code/core/system/proxy-camera/src/OpenCVCamera.cpp

```diff
@@ -83,6 +84,9 @@ namespace proxy {
             bool retVal = false;
 
             if ( (dest != NULL) && (size > 0) && (m_image != NULL) ) {
+                if(m_flipped){
+                    cvFlip(m_image, m_image, -1);
```

Comment:
@HangYin2015 Does the flipping work "in-place", i.e. without the need of a temporary image?